### PR TITLE
Maintainer terms and guidelines

### PR DESCRIPTION
**Pull request by jasperweyne**
This commit adds an initial version for the maintainer terms and
guidelines. It includes a ruleset for the maintainers to adhere to,
intended to create a level playing field for all maintainers.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+any other. Therefore, a group of people (hereafter referred to as organisation) 
+may apply for maintainer status. With maintainer status, your organisation will
+be able to weigh in on the direction and priorities of the project. In return, 
+and in the spirit of free, open source software, we ask that your organisation 
+actively participates in the development and maintenance of the project. (Note: 
+individuals that are not affiliated to any organisation are welcome to apply 
+for maintainer status too.)
+
+## Terms of maintainer status
+Each entity with maintainer status shall provide one or more maintainers, who'll
```
Comment (A-Daneel) [inline MAINTAINERS.md:22]: please, change this "who'll" to "who will"

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+groups in all shapes and sizes. As such, it's continously developed by these 
+different groups of people. As any open source project alike, open source is
+about collaboration. However, different users may have different needs and
+different priorities. To bring these differing views together, the following
+terms and guidelines were devised to bring everyone together on equal grounds.
+
+This project is intended for use by organised groups of people: associations
+organisations, sports clubs or anything alike. Regardless of group size, each
+of these entities has a unique perspective that should be weighed equally to 
+any other. Therefore, a group of people (hereafter referred to as organisation)
```
Comment (A-Daneel) [inline MAINTAINERS.md:13]: in this sentence it really should be "an organisation"

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  breaking BC. Any BC breaks you plan on merging MUST be discussed with the
+  maintainer team prior to merging.
+* Is this a new feature? If so:
+    * If no issue is linked, does the PR contain information indicating the need
+      for the feature? If not, ask them to provide that information, preferably
+      in an issue.
+    * Are the new tests actually meaningful? Do they sensibly test the behaviour,
+      similarly to how a user would experience it? If not, **do not merge** the
+      feature until they are!
+    * Is the feature necessary for general use cases? Kiwi isn't, nor should it
```
Comment (A-Daneel) [inline MAINTAINERS.md:116]: like we talked about, this sentence could do with a rewrite to make it easier to read

File: MAINTAINERS.md (hunk 1 above)
Comment (A-Daneel) [inline MAINTAINERS.md:22]: please, change this "who'll" to "who will"

File: MAINTAINERS.md (hunk 2 above)
Comment (A-Daneel) [inline MAINTAINERS.md:13]: in this sentence it really should be "an organisation"

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  clarification reveal that the author may be unaware of features that already
+  accommodate their needs, show an example on how to address the issue with the
+  current features and feel free to close the issue once they confirm.
+* Does the change break backwards compatibility (BC)? In general, no data should
+  be lost. If so, work with the contributor to determine why the BC break is
+  needed, and whether or not there may be a way to make the change without
+  breaking BC. Any BC breaks you plan on merging MUST be discussed with the
+  maintainer team prior to merging.
+* Is this a new feature? If so:
+    * If no issue is linked, does the PR contain information indicating the need
```
Comment (A-Daneel) [inline MAINTAINERS.md:110]: I think this should be "indicate"

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+representing a single entity are present, only of them may cast a vote on behalf
+of the entity they represent. For any changes to these terms and guidelines, as
+well as the contribution guidelines and the code of conduct, a vote must be held.
+
+Maintainers are expected to be active, supportive and responsive. Upon questions
+or requests from fellow maintainers, a response or indication of acknowledgement
+of the request must be given within 72 hours. If a maintainer is unavailable for
+a longer period of time, a notice to must be provided accordingly. Maintainers
+are expected to adhere to all deadlines and agreements. General development
+meetings (whose schedule is decided upon during votes) shall be regularly
```
Comment (A-Daneel) [inline MAINTAINERS.md:45]: could be that I am wrong here, is "whose schedule" the correct phrasing?

**Review by A-Daneel — DISMISSED** [review]
We could definitely merge this. If you could resolve these small points of attention (spelling and grammar mostly) it would be lovely, but even without it's a nice and concise document.

File: MAINTAINERS.md (hunk 5 above)
Comment (jasperweyne) [reply MAINTAINERS.md:45]: [Arguably](https://en.wikipedia.org/wiki/Inanimate_whose)

**Pull request merged**